src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+export const BOOK_ACTION = {
+  ADD: 'add_book',
+  REMOVE: 'remove_book',
+};
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20action%20types%20for%20adding%20and%20removing%20a%20book.), Please Define action types for adding and removing a book using [ducks pattern](https://github.com/erikras/ducks-modular-redux).
[Hint]: For example: `const ADD_BOOK = bookstore/books/ADD_BOOK`